Java/TelemetrySystem/src/test/java/tddmicroexercises/telemetrysystem/TelemetryDiagnosticControlsTest.java

```diff
@@ -1,13 +1,31 @@
 package tddmicroexercises.telemetrysystem;
 
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import tddmicroexercises.telemetrysystem.services.impl.ClientImpl;
+import tddmicroexercises.telemetrysystem.services.impl.ConnectionImpl;
+import tddmicroexercises.telemetrysystem.services.impl.TelemetryDiagnosticControls;
 
-public class TelemetryDiagnosticControlsTest
-{
-    
-	@Test
-    public void CheckTransmission_should_send_a_diagnostic_message_and_receive_a_status_message_response()
-    {
+
+public class TelemetryDiagnosticControlsTest {
+    public TelemetryDiagnosticControlsTest() {
+    }
+
+    @Test
+    public void CheckTransmission_should_send_a_diagnostic_message_and_receive_a_status_message_response() throws Exception {

```

Comment:
add few more test cases. don't write all the testcases related to client, connection, controls in a single method, segregate them into different methods. Also follow Arrange-act-asset pattern while writing testcases

Reply:
Worked on feedback